tests/subsys/debug/coredump_backends/src/main.c

```diff
@@ -45,7 +45,7 @@ static void check_errors(void)
 	}
 }
 
-void test_coredump(void)
+static void *coredump_setup(void)
```

Comment:
This is an actual test, and not just setting up the environment.

Reply:
Yes, It is a test, but the other two tests' result depends on this test to run firstly. So I move this test to setup.

Reply:
Without able to define the sequence, the new version is probably the easiest way to achieve this.